### DISCUSSION TITLE
Plugin doesn't create cards but message that everything is ok

### DISCUSSION THREAD

**Discussion by VadymIgnatiev** [ObsidianToAnki/Obsidian_to_Anki#264] [Q&A]
I installed this plugin according to this page:
https://github.com/Pseudonium/Obsidian_to_Anki/wiki/Steps-for-new-users
So I have Anki, I added AnkiConnect with appropriate settings like here:
![image](https://user-images.githubusercontent.com/37376422/126178996-c1d704dc-3179-4c3d-8ef7-d1433f1c7e26.png)
And I install the plugin. So for test purposes, I create a card with the next content:

![image](https://user-images.githubusercontent.com/37376422/126180588-2cdeff4c-4a93-4b4d-8b75-1ca93d3c65ad.png)

When I click "Scan vautl" button I have the next messages:
![image](https://user-images.githubusercontent.com/37376422/126179836-54741745-0362-41c1-8874-b2c3ea2a0536.png)

So, as I understand, everything is good, but in reality, the plugin doesn't create the card in the Anki, it even doesn't create a tag with a message-id. What happened, what is wrong? Because I have no idea. Thanks for any help.

**Comment by alexandermoennig** [comment]
I had the same issue today. I had to translate the card type and Front/Back to the language set in anki.
